sections/collection-template.liquid

```diff
@@ -84,8 +84,26 @@
                       <select class="filters-toolbar__input hidden" name="FilterTags" id="FilterTags" aria-describedby="a11y-refresh-page-message a11y-selection-message" data-select-input>
                         <option value="{{ collection.url }}">{{ 'collections.filters.all_tags' | t }}</option>
                         {% for tag in collection.all_tags %}
-                          {%- assign new_url = tag | link_to_tag: tag | split: 'href="' | last | split: '"' | first -%}
-                          <option value="{{ new_url }}"{% if current_tags contains tag %} selected="selected"{% endif %}>{{ tag }}</option>
+                        
+<!--                     removing clitoral and vaginal from only for him collection -->
+ 						 {% if tag !='clitoral'%}
+                        	{% if tag !='vaginal'%}
```

Comment:
You can combine these conditions... Plus these are running on every collection page..